frontend/simple/assets/sass/_groupincome.scss

```diff
@@ -27,4 +27,14 @@
     overflow: hidden;
     text-overflow: ellipsis;
   }
+
+  &fade-enter-active,
+  &fade-leave-active {
+    transition: opacity 0.5s;
+  }
+
+  &fade-enter,
+  &fade-leave-to {
+    opacity: 0;
+  }
```

Comment:
Is this CSS being used? Where?

Reply:
see my last comment:
> for now, I did a temporary fade-out animation, as the more complex solution would need work done in #426 . capturing this todo part there too.

Reply:
So, that means we can remove this CSS for now, correct?

Reply:
Oh okay, maybe we should write that css in a separated file, what do u think?